tests/phpunit/tests/post/primePostCaches.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Test `_prime_post_caches()`.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Test class for `_prime_post_caches()`.
+ *
+ * @group post
+ *
+ * @covers ::_prime_post_caches
+ */
+class Tests_Post_PrimePostCaches extends WP_UnitTestCase {
+
+	/**
+	 * Post IDs.
+	 *
+	 * @var int[]
+	 */
+	public static $posts;
+
+	/**
+	 * Set up test resources before the class.
+	 *
+	 * @param WP_UnitTest_Factory $factory The unit test factory.
+	 */
+	public static function wpSetupBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$posts = $factory->post->create_many( 3 );
+
+		$category = $factory->term->create(
+			array(
+				'taxonomy' => 'category',
+				'slug'     => 'foo',
+				'name'     => 'Foo',
+			)
+		);
+
+		wp_set_post_terms( self::$posts[0], $category, 'category' );
+		add_post_meta( self::$posts[0], 'meta', 'foo' );
+		add_post_meta( self::$posts[1], 'meta', 'bar' );
+	}
+
+	public function set_up() {
+		parent::set_up();
+
+		foreach ( self::$posts as $post_id ) {
+			clean_post_cache( $post_id );
+		}
+	}
+
```

Comment:
```suggestion
```

This isn't needed as cache is flushed as part of the global scope cleanup in the default setup.

https://github.com/WordPress/wordpress-develop/blob/64e842e7f932acbec383508d6aa177832508b14f/tests/phpunit/includes/abstract-testcase.php#L117

https://github.com/WordPress/wordpress-develop/blob/64e842e7f932acbec383508d6aa177832508b14f/tests/phpunit/includes/abstract-testcase.php#L202-L207

https://github.com/WordPress/wordpress-develop/blob/64e842e7f932acbec383508d6aa177832508b14f/tests/phpunit/includes/abstract-testcase.php#L368-L380